src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
-2.  Look up the master version and upgrade manifest, which is saved within the CLI docker image in the /version subfolder.
-3.  Perform initialization and configuration of Che such as with `eclipse/che-init`. This image contains templates to be installed onto your computer used by the CLI to configure Che for your specific OS.
+* Perform initialization and configuration of Che as done with the `eclipse/che-init` command. This image contains templates to be installed on your computer used by the CLI to configure Che for your specific OS.
 
-You can control how Che downloads these images with command line options. All image downloads are performed with `docker pull`.
+You can control how Che downloads these images with command line options. All image downloads are performed using the `docker pull` command.
 
 [width="100%",cols="32%,68%",options="header",]
 |===
-|Mode >>>>>>>>>> |Description
-|`--no-force` |Default behavior. Will download an image if not found locally. A local check of the image will see if an image of a matching name is in your local registry and then skip the pull if it is found. This mode does not check DockerHub for a newer version of the same image.
-|`--pull` |Will always perform a `docker pull` when an image is requested. If there is a newer version of the same tagged image at DockerHub, it will pull it, or use the one in local cache. This keeps your images up to date, but execution is slower.
-|`--force` |Performs a forced removal of the local image using `docker rmi` and then pulls it again (anew) from DockerHub. You can use this as a way to clean your local cache and ensure that all images are new.
-|`--offline` |Loads Docker images from `backup/*.tar` folder during a pre-boot mode of the CLI. Used if you are performing an installation or start while disconnected from the Internet.
+|Mode |Description
+|`--no-force` |Default behavior. Downloads an image if not found locally. A local check of the image inspects if an image of a matching name is present in your local registry and then skips pulling the image if it is found. This mode does not check Docker Hub for a newer version of the same image.
+|`--pull` |Always perform a `docker pull` command when an image is requested. If there is a newer version of the same tagged image at Docker Hub, it pulls it, or uses the one in the local cache. This slows the execution, but keeps your images up-to-date.
+|`--force` |Performs a forced removal of the local image using the `docker rmi` command and then pulls it again from Docker Hub. Use this to clean your local cache and to ensure that all images are new.
+|`--offline` |Loads Docker images from the `backup/*.tar` directory during a pre-boot mode of the CLI. Used if you are performing an installation or start while disconnected from the Internet.
```

Comment:
I doubt that `*.tar` is a directory. Based on what's written below, we could rewrite this to make it more accurate:

"Loads `tar`-archived Docker images from the `backup` directory during the pre-boot mode of the CLI."